parser/feed/src/main/java/de/danoeh/antennapod/parser/feed/PodcastIndexTranscriptParser.java

```diff
@@ -0,0 +1,191 @@
+package de.danoeh.antennapod.parser.feed;
```

Comment:
How about moving this to a new module `parser/transcript`? It's not really parsing feeds. Then future transcript formats are all in one place.

Reply:
done.  Let me know if you wanted it further up in $ROOT/parser/transcript

Reply:
I would indeed prefer `$ROOT/parser/transcript`, so using a new module. The reason is that modules prevent cyclic dependencies, while packages still allow cyclic dependencies (see also #4661)

Reply:
done with $ROOT/parser/transcript